.github/workflows/golang.yaml

```diff
@@ -30,26 +30,38 @@ jobs:
     steps:
     - uses: actions/checkout@v4
       name: Checkout code
+    - name: Get Golang version
+      id: vars
+      run: |
+        GOLANG_VERSION=$( grep "GOLANG_VERSION ?=" versions.mk )
+        echo "GOLANG_VERSION=${GOLANG_VERSION##GOLANG_VERSION ?= }" >> $GITHUB_ENV
     - name: Install Go
       uses: actions/setup-go@v5
       with:
-        go-version-file: 'go.mod'
+        go-version: ${{ env.GOLANG_VERSION }}
```

Comment:
@ArangoGutierrez we should probably look at how we can reuse something like this.

Reply:
for all other projects, yup